src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -453,7 +453,8 @@ internal static bool IsEdgeSupportedType(Type type)
             {
                 // since HTTP headers are case insensitive, we lower-case the keys
                 // as does Node.js request object
-                headers.Add(header.Key.ToLowerInvariant(), header.Value.First());
+                string values = string.Join(" ", header.Value);
```

Comment:
Yeah, I think commas are correct as per spec https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html. Ideally I'd be able to just get the raw unleaded value from WebApi, but I haven't found out how yet (if possible at all).